{"Launchpad":"https://bugs.launchpad.net/bugs/1510150","Id":"1510150","Gerrit":"https://review.openstack.org/239517","IdGerrit":"239517","IdCommit":"64172b01cd5d8eda3aa282262d7ca060066161d7","Files":["cinder/tests/unit/test_nfs.py","cinder/volume/drivers/nfs.py"],"Lines_Inseted":8,"Lines_Deleted":3,"CommitParent":"321c9c901d3b456c188a5e76f4dd102bdde2c0d8","Classification":"Not_Bug","KeywordsTitle":"NfsException, not installed","Description":"check, OSError, errno, NfsException","KeywordsCommit":"check","Comments":""}